### DISCUSSION TITLE
Locale calendars

### DISCUSSION THREAD

**Discussion by peimn** [statamic/cms#9784] [General]
In addition to the Gregorian calendar, there are other significant calendars in use, such as the lunar Hijri and the solar Hijri. Lunar time is employed in Arab countries, while Persian countries use solar time.

I've enhanced the VCalendar to accommodate local dates, including these two calendars. However, development of the calendar by its original author appears to have halted, with the last response to an issue on the repository being several months ago.

While I plan to submit a PR to the calendar, I'm also contemplating republishing it as a separate package. This would provide complete control over its development and updates. Furthermore, By maintaining a separate package, timely updates and enhancements can be ensured, independent of the responsiveness of the original calendar's author.

Any thoughts?

**Comment by jasonvarga** [comment]
That's a shame. These situations are frustrating. I'd rather not use a fork. Let's see if they merge your PR soon. It looks like PRs to the base branch get merged reasonably quickly. If nothing happens for a little while we can consider changing to a fork.

**Reply by peimn** [reply]
Okay, I have a few tasks on my plate, such as creating demos and updating documentation. Currently, the calendars are functioning correctly, but we need to conduct more tests to ensure that all use cases are covered.
